src/js/index.js

```diff
@@ -35,7 +35,7 @@ function startFormService() {
     });
 }
 
-function startAuth(isFormService) {
+function startAuth() {
```

Comment:
Just removed an unused function parameter. Forgot to remove it in the last commit.